sys-process/wait_on_pid/wait_on_pid-0.0.2-r1.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI="8"
```

Comment:
Now this does require a revbump and dropping the keywords. Then I see the ebuild has more cleaning that we can do while updating it. 

But is this package still functional? Should it be kept around?

Reply:
That's what I thought last time.
Is it functional? - I don't know, it's pulled in as a dependency for media-tv/gentoo-vdr-scripts-3.0_rc2-r1.
Should it be kept around - to be honest I don't know if it's that useful.
Does it work with EAPI="8"? Yes, just tested it.

I'm completely new to making these types of changes, just figuring out how to make this pull request took me 2 hours.
Thanks for letting me know, that this actually requires a revbump.